osu.Game.Tournament/Components/DrawableTournamentTeam.cs

```diff
@@ -14,15 +12,15 @@ namespace osu.Game.Tournament.Components
 {
     public abstract partial class DrawableTournamentTeam : CompositeDrawable
     {
-        public readonly TournamentTeam Team;
+        public readonly TournamentTeam? Team;
 
         protected readonly Container Flag;
         protected readonly TournamentSpriteText AcronymText;
 
         [UsedImplicitly]
-        private Bindable<string> acronym;
+        private Bindable<string>? acronym;
```

Comment:
why? it's initialised in BDL. should be non-nullable and `= null!` instead

Reply:
if `Team` is null BDL will early return, and acronym will be null